source/patterns/@aws-solutions-constructs/core/lib/cloudfront-distribution-helper.ts

```diff
@@ -125,13 +125,14 @@ export function CloudFrontDistributionForS3(scope: cdk.Construct,
 
   // Extract the CfnBucketPolicy from the sourceBucket
   const bucketPolicy = sourceBucket.policy as s3.BucketPolicy;
-  addCfnSuppressRules(bucketPolicy, [
-    {
-      id: 'F16',
-      reason: `Public website bucket policy requires a wildcard principal`
-    }
-  ]);
-
+  if (bucketPolicy) {
```

Comment:
Let's add a comment explaining that the lack of a bucketPolicy means the bucket was imported from outside the stack so the lack of cfn_nag suppression is not an issue.